app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -1647,6 +1648,15 @@ class PluginApi {
     addPluginOutletDecorator(outletName, callback, opts || {});
   }
 
+  /**
+   * Used to set the min and max height for the topic timeline scroll area. Pass object with min/max key value pairs.
```

Comment:
maybe you can throw in the desktop keyword for clarity?

Reply:
Ohh nice. Yeah.